jax/experimental/sparse/transform.py

```diff
@@ -875,9 +875,14 @@ def fun(*arrs):
   outvals = jax.custom_derivatives.custom_jvp_call_p.bind(fun, jvp, *invals, **params)
   return arrays_to_spvalues(spenv, outvals)
 
-sparse_rules_bcoo[jax.custom_derivatives.custom_jvp_call_p] = _custom_jvp_sparse_rule
-sparse_rules_bcsr[jax.custom_derivatives.custom_jvp_call_p] = _custom_jvp_sparse_rule
+sparse_rules_bcoo[jax.custom_derivatives.custom_jvp_call_p] = _custom_jvp_call_sparse_rule
+sparse_rules_bcsr[jax.custom_derivatives.custom_jvp_call_p] = _custom_jvp_call_sparse_rule
 
+def _custom_vjp_call_jaxpr_sparse_rule(spenv, *spvalues, fun_jaxpr, **params):
+  return eval_sparse(fun_jaxpr.jaxpr, fun_jaxpr.consts, spvalues, spenv)
+
+sparse_rules_bcoo[jax.custom_derivatives.custom_vjp_call_jaxpr_p] = _custom_vjp_call_jaxpr_sparse_rule
+sparse_rules_bcsr[jax.custom_derivatives.custom_vjp_call_jaxpr_p] = _custom_vjp_call_jaxpr_sparse_rule
```

Comment:
Note: this is required for the sparse doctests, which run `sparsify(optimize.minimize)`, because `jaxopt` uses custom_vjp. Since it's a small change, I thought I'd do it as part of this PR.